README.md

```diff
@@ -9,6 +9,14 @@ This repository contains Cloud One File Storage Security post-scan actions refer
 1. Go to the [post-scan-actions](post-scan-actions/) or [scan-triggers](scan-triggers/) folder to view all the example Lambda functions.
 2. Follow the instructions in the readme of the chosen example to deploy it.
 
+#### Plugins Available on Serverless Application Repository
+
+Some plugins are hosted in Serverless Application Repository and you can find them with the following steps.
+
+1. Go to Serverless Application Repository > Available applications > Public applications.
```

Comment:
1. From AWS, go to Serverless Application Repository > Available applications > Public applications.